server/routes/mpesa.py

```
@@ -0,0 +1,367 @@
+        # Validate required parameters
+        if not all([amount, phone_number]):
+            return jsonify({
+                'success': False,
+                'message': 'Missing required parameters: amount, phone_number'
+            }), 400
+
+        # Convert amount to integer and validate
+        try:
+            amount = float(amount)
+            if amount <= 0:
+                raise ValueError("Amount must be positive")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)